setup.py

```diff
@@ -25,7 +25,8 @@
 # Dependencies
 extras_require = {
     'serial':   ['pyserial~=3.0'],
-    'neovi':    ['python-ics>=2.12']
+    'neovi':    ['python-ics>=2.12'],
+    'mf4':      ['asammdf>=5.5.0;python_version>="3.6"', 'numpy>=1.16.0;python_version>="3.6"']
```

Comment:
By now, the `;python_version>="3.6"` can be removed, since we do not support Python < 3.6 anyways.

Reply:
@felixdivo  
done